pkg/timeutils/utils_test.go

```diff
@@ -0,0 +1,24 @@
+package timeutils
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+func TestGetTimestamp(t *testing.T) {
+	now := time.Now()
+	cases := []struct{ in, expected string }{
+		{"0", "-62167305600"},
+		{"2015-12-05", "1449273600"},
+		{"invalid", "invalid"},
+		{"1m", fmt.Sprintf("%d", now.Add(-1*time.Minute).Unix())},
```

Comment:
I think we need more testing here.  In particular, timestamps with Z's and Z-07:00 type of syntax.